README updates (#25906)
DevExpress/DevExtreme: README.md

- Angular
    - [Create a DevExtreme Angular Application](https://js.devexpress.com/Angular/Documentation/Guide/Angular_Components/Getting_Started/Create_a_DevExtreme_Application/)
    - [Add DevExtreme to an Angular Application](https://js.devexpress.com/Angular/Documentation/Guide/Angular_Components/Getting_Started/Add_DevExtreme_to_an_Angular_CLI_Application/)

- React
    - [Create a DevExtreme React Application](https://js.devexpress.com/React/Documentation/Guide/React_Components/Create_a_DevExtreme_Application/)
    - [Add DevExtreme to a React Application](https://js.devexpress.com/React/Documentation/Guide/React_Components/Add_DevExtreme_to_a_React_Application/)

- Vue
    - [Create a DevExtreme Vue Application](https://js.devexpress.com/Vue/Documentation/Guide/Vue_Components/Create_a_DevExtreme_Application/)
    - [Add DevExtreme to a Vue Application](https://js.devexpress.com/Vue/Documentation/Guide/Vue_Components/Add_DevExtreme_to_a_Vue_Application/)

- jQuery
    - [Add DevExtreme to a jQuery Application](https://js.devexpress.com/jQuery/Documentation/Guide/jQuery_Components/Add_DevExtreme_to_a_jQuery_Application/)

> ## Learn

Reviewer: I would also add an entry for our new UI Templates: https://js.devexpress.com/Templates/UITemplates/

- [Component Demos](https://js.devexpress.com/Demos/WidgetsGallery/)

> - [Documentation](https://js.devexpress.com/Documentation)

Reviewer: Trailing slash is absent in URL

# DevExtreme

> ![DevExtreme components](https://js.devexpress.com/Content/Images/Main/modern/AdaptivityDesktop.png)

Reviewer 2: It's better to place the DevExtreme description before the big screenshot.
I think we can use the previous one
```
DevExtreme is a set of enterprise-ready UI component suites for Angular, React, Vue, and jQuery. It is everything you need to create responsive web apps for touch devices and traditional desktops: data grid, interactive charts, data editors, navigation and multi-purpose widgets. These controls are designed to look great and to provide powerful functionality in any browser.
```

Reviewer 2: Or we can use your rewritten text. But I'm not sure if several paragraphs suit here

> DevExtreme is an enterprise-ready suite of powerful, good-looking UI components for popular front-end frameworks: Angular, React, Vue, and jQuery.

Reviewer 3: ```suggestion
DevExtreme is an enterprise-ready suite of powerful, engaging UI components for popular front-end frameworks: Angular, React, Vue, and jQuery.
```

DevExtreme is an enterprise-ready suite of UI components for popular front-end frameworks: Angular, React, Vue, and jQuery.

DevExtreme components are responsive and accessible. They work well across different devices, screen sizes, and input methods. Whether your target audience uses phones, PCs or screen readers - DevExpress components are up to the task.

Our range of powerful, modern JavaScript components includes data grids, interactive charts, data editors and widgets. View the [Demo Gallery](https://js.devexpress.com/Demos/WidgetsGallery/) to see our components in action.

- Official website: [js.devexpress.com](https://js.devexpress.com)
- Pricing: [js.devexpress.com/buy](https://js.devexpress.com/Buy)

> - Licensing FAQ: [js.devexpress.com/licensing](https://js.devexpress.com/Licensing)

Reviewer 2: Let's leave the name `Licensing` because there are links to all licensing docs and FAQ (not only FAQ)

- Support: [www.devexpress.com/support](https://www.devexpress.com/support)

## Getting Started

> The DevExtreme website includes "Getting Started" guides for every flavor of DevExtreme:

Reviewer 3: ```suggestion
The DevExtreme website includes "Getting Started" guides for every aspect of DevExtreme:
```

- Angular
    - [Create a DevExtreme Angular Application](https://js.devexpress.com/Angular/Documentation/Guide/Angular_Components/Getting_Started/Create_a_DevExtreme_Application/)
    - [Add DevExtreme to an Angular Application](https://js.devexpress.com/Angular/Documentation/Guide/Angular_Components/Getting_Started/Add_DevExtreme_to_an_Angular_CLI_Application/)

- React
    - [Create a DevExtreme React Application](https://js.devexpress.com/React/Documentation/Guide/React_Components/Create_a_DevExtreme_Application/)
    - [Add DevExtreme to a React Application](https://js.devexpress.com/React/Documentation/Guide/React_Components/Add_DevExtreme_to_a_React_Application/)

- Vue
    - [Create a DevExtreme Vue Application](https://js.devexpress.com/Vue/Documentation/Guide/Vue_Components/Create_a_DevExtreme_Application/)
    - [Add DevExtreme to a Vue Application](https://js.devexpress.com/Vue/Documentation/Guide/Vue_Components/Add_DevExtreme_to_a_Vue_Application/)

- jQuery
    - [Add DevExtreme to a jQuery Application](https://js.devexpress.com/jQuery/Documentation/Guide/jQuery_Components/Add_DevExtreme_to_a_jQuery_Application/)

## Learn

> - [Component Demos](https://js.devexpress.com/Demos/WidgetsGallery)

Reviewer 2: We already call them [Demo Gallery](https://js.devexpress.com/Demos/WidgetsGallery/) above
Maybe it's better to stick to this term

> - [Ready-To-Run Application Templates](https://js.devexpress.com/Templates/UITemplates)

Reviewer 2: The are not applications templates (and it can be confusing, because we have [Application templates](https://js.devexpress.com/React/Documentation/Guide/React_Components/Application_Template/) and we can even include them in the list)

We call them `Responsive UI Templates` everywhere, let's use this term here as well
https://js.devexpress.com/Angular/Templates/UITemplates/

Reviewer 2: > [YouTube videos](https://www.youtube.com/user/DeveloperExpress/)

We have a list of DevExtreme videos on YouTube
https://www.youtube.com/playlist?list=PL8h4jt35t1wjGvgflbHEH_e3b23AA30-z

- [Documentation](https://js.devexpress.com/Documentation)
- [Examples on GitHub](https://github.com/DevExpress/DevExtreme-examples)
- [YouTube videos](https://www.youtube.com/playlist?list=PL8h4jt35t1wjGvgflbHEH_e3b23AA30-z)

## Contributing

> The DevExtreme team welcomes outside contributions - bug reports and feature requests. Read our [contribution guidelines](CONTRIBUTING.md) for more information.

Reviewer 3: ```suggestion
The DevExtreme team welcomes outside contributions of bug reports and feature requests. Read our [contribution guidelines](CONTRIBUTING.md) for more information.
```
